discovery-infra/utils.py

```diff
@@ -240,6 +240,14 @@ def file_exists(file_path):
     return Path(file_path).exists()
 
 
+def create_folder(folder, with_chmod=True):
```

Comment:
same code as here recreate_folder? just add here flag "recreate if exists" and delete recreate_folder